apps/aechannel/src/aesc_fsm.erl

```diff
@@ -2199,37 +2199,36 @@ new_onchain_tx_for_signing(Type, Opts, D) ->
     end.
 
 new_onchain_tx_for_signing_(Type, Opts, D) ->
-    FeeSpecified = maps:is_key(fee, Opts),
     Defaults = tx_defaults(Type, Opts, D),
     Opts1 = maps:merge(Defaults, Opts),
-    {ok, Tx, Updates} = new_onchain_tx(Type, Opts1, D),
-    CurHeight = cur_height(),
-    case {aetx:min_fee(Tx, CurHeight), aetx:fee(Tx)} of
+    CurrHeight = cur_height(),
```

Comment:
```
$ grep -r -o -e "cur_height" apps | wc -l 
4
 $ grep -r -o -e "curr_height" apps | wc -l
0
 $ grep -r -o -e "CurrHeight" apps | wc -l 
44
 $ grep -r -o -e "CurHeight" apps | wc -l 
13
```
There should be a ticket to make the variable names consistent.

Reply:
yes, we are rather inconsistent in that regard

Reply:
Since almost all occurrences were of `CurrHeight`, I've replaced all `CurHeight` accordingly.